userland/shell/sh.c

```
@@ -334,13 +360,33 @@ int main(int argc, char *argv[]) {
     signal(SIGINT, SIG_IGN);
     pq_init(&g_pool.pq);
     g_pool.shutting_down = 0;
-    pthread_mutex_init(&g_pool.mutex, NULL);
-    pthread_cond_init(&g_pool.cond, NULL);
+    if (pthread_mutex_init(&g_pool.mutex, NULL) != 0) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)